web-ui/src/main/resources/catalog/components/viewer/geometry/GeometryService.js

```diff
@@ -307,11 +308,17 @@
                 '<gml:feature xmlns:gml="http://www.opengis.net/gml">' +
                 '<geometry>' +
                 input + '</geometry></gml:feature></featureMembers>';
-            var feature = format.readFeatures(fullXml, {
-              dataProjection: options.crs,
-              featureProjection: outputProjection
-            })[0];
-            outputValue = feature.getGeometry();
+
+            try {
```

Comment:
There is no reason for a try/catch block here. The `parseGeometryInput` *will* throw errors if encountered and it is the responsability of the one using it to catch these accordingly. This is done here for example:
https://github.com/geonetwork/core-geonetwork/blob/develop/web-ui/src/main/resources/catalog/components/edit/bounding/BoundingDirective.js#L243
Making the error go straight to the console means we can't use it for something else, i.e. display in the UI.

(I agree that this might be nice to have in the doc block.)

I guess a try/catch block could be added here as well:
https://github.com/geonetwork/core-geonetwork/blob/develop/web-ui/src/main/resources/catalog/components/edit/bounding/BoundingDirective.js#L173